kolibri/core/assets/src/api-resources/contentNode.js

```diff
@@ -48,6 +48,37 @@ export default class ContentNodeResource extends Resource {
     }
     return promise;
   }
+
+  getCopies(id) {
+    if (!id) {
+      throw TypeError('An id must be specified');
+    }
+    let promise;
+    this.copies_cache = this.copies_cache || {};
+    const key = this.cacheKey({ id });
```

Comment:
Interested to know more generally your thoughts on this, and whether it would be helpful to have something that we could use as a drop in, rather than repeating this boilerplate?

Reply:
I do agree something more reusable would be a cleaner solution.

Reply:
maybe open this up as follow-up issue?